src/main/java/org/primefaces/model/menu/DefaultMenuItem.java

```diff
@@ -69,36 +69,26 @@
     private boolean escape = true;
     private String rel;
 
-    public DefaultMenuItem() {
-    }
-
-    public DefaultMenuItem(Object value) {
-        this.value = value;
-    }
-
-    public DefaultMenuItem(Object value, String icon) {
-        this.value = value;
-        this.icon = icon;
-    }
-
-    public DefaultMenuItem(Object value, String icon, String url) {
-        this.value = value;
-        this.icon = icon;
-        this.url = url;
+    private DefaultMenuItem() {
```

Comment:
To not break compatibility I'd prefer to keep the public CTORs as well, maybe setting them `@Deprecated`. But that actually depends if this feature will be included in a patch or minor release?